docs/audit.md

```diff
@@ -1,3 +1,18 @@
+<!-- START doctoc generated TOC please keep comment here to allow auto update -->
+<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
+
+-   [Audit](#audit)
+    -   [How to Audit a Baseline](#how-to-audit-a-baseline)
+        -   [Windows Powershell and cmd](#windows-powershell-and-cmd)
+        -   [Windows git bash](#windows-git-bash)
+        -   [MacOS & Linux](#macos--linux)
+    -   [Manually Labelling Secrets](#manually-labelling-secrets)
+        -   [Handling Developer Secrets](#handling-developer-secrets)
+    -   [What to do after marking an potential secret as a valid secret?](#what-to-do-after-marking-an-potential-secret-as-a-valid-secret)
+    -   [Comparing Baselines](#comparing-baselines)
+
+<!-- END doctoc generated TOC please keep comment here to allow auto update -->
```

Comment:
Would you mind moving the TOC below `# Audit` and regenerating it? I'm suggesting this because the page name will already be seen by users at the top so it's redundant to include it in the TOC and our other docs already follow this standard.

Reply:
Good point! fixed 51479d1